core/src/test/kotlin/ch/ergon/dope/UnnestClauseTest.kt

```diff
@@ -0,0 +1,71 @@
+package ch.ergon.dope
+
+import ch.ergon.dope.helper.someArrayField
+import ch.ergon.dope.helper.someBucket
+import ch.ergon.dope.helper.someStringField
+import ch.ergon.dope.resolvable.expression.alias
+import ch.ergon.dope.resolvable.expression.unaliased.type.asParameter
+import ch.ergon.dope.resolvable.expression.unaliased.type.logical.and
+import ch.ergon.dope.resolvable.expression.unaliased.type.relational.isEqualTo
+import ch.ergon.dope.resolvable.expression.unaliased.type.relational.isValued
+import ch.ergon.dope.resolvable.fromable.asterisk
+import junit.framework.TestCase.assertEquals
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+
+class UnnestClauseTest {
+    private lateinit var create: QueryBuilder
+
+    @BeforeTest
+    fun setup() {
+        create = QueryBuilder()
+    }
+
+    @Test
+    fun `should support nested unnest`() {
+        val airline = someBucket("airline").alias("ai")
+        val aField = someArrayField("a")
+        val expected = "SELECT c FROM airline AS ai UNNEST a AS ab UNNEST ab.c WHERE a IS VALUED"
+
+        val actual: String = create
+            .select(someArrayField("c"))
+            .from(airline)
+            .unnest(aField.alias("ab"))
+            .unnest(someArrayField("c", someBucket("ab")))
+            .where(aField.isValued())
+            .build()
+
+        assertEquals(expected, actual)
+    }
+
+    @Test
```

Comment:
Add a test for `SELECT * FROM b UNNEST ARRAY_CONCAT(["a"], ["a"]) as a` as well (or if there is no function of type array, make sure that it will work in the future)